CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master / unreleased
 
+## 1.2.0 / 2020-06-xx
```

Comment:
Aren't we doing -rc.0 first?

Reply:
I think it should be `-rc.0` and the date should be the one of the RC release (ie. today).

Reply:
Sorry, it's not clear in the instructions. "Update the version number in the VERSION file".
I'll see if I can improve the instructions.

Reply:
OK I re-read the instructions and now I don't know what to do.  It clearly says I should merge this PR before tagging `-rc.0`, but it doesn't say what VERSION should contain at this point.

Reply:
In last releases, we did two PRs: one (or more) for RC, one for final release. Each time merging to release branch. After final release, release branch is merged (without squashing) to master.

Reply:
Do you have any suggestion what VERSION should contain before I do any of that?

Reply:
VERSION should also be 1.2.0-rc.0 at first.

Reply:
Sorry, didn't realise we were in CHANGELOG here.
I don't see the point in putting -rc.x in CHANGELOG, just in VERSION.